plugins/inputs/huawei_routers_telemetry/huawei_routers_telemetry.go

```diff
@@ -0,0 +1,230 @@
+package huawei_routers_telemetry
+
+import (
+	"fmt"
+	"github.com/DamRCorba/huawei_telemetry_sensors"
+	"github.com/DamRCorba/huawei_telemetry_sensors/sensors/huawei-telemetry"
+	"github.com/golang/protobuf/proto"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/metric"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"io"
+	"net"
+	"os"
+	"strings"
+	"sync"
+	"time"
```

Comment:
Please group this for standard libraries, telegraf and other...
```suggestion
	"fmt"
	"io"
	"net"
	"os"
	"strings"
	"sync"
	"time"

	"github.com/DamRCorba/huawei_telemetry_sensors"
	"github.com/DamRCorba/huawei_telemetry_sensors/sensors/huawei-telemetry"

	"github.com/golang/protobuf/proto"

	"github.com/influxdata/telegraf"
	"github.com/influxdata/telegraf/internal"
	"github.com/influxdata/telegraf/metric"
	"github.com/influxdata/telegraf/plugins/inputs"
```

Reply:
i made the change. I pushing in when i finish te review the other points